Create inline_link_in_paragraph_is_distinguishable_ be4d0c.md (#1010)
act-rules/act-rules.github.io: pages/glossary/different-hue.md

---
title: Different hue
key: different-hue
unambiguous: true
objective: true
---

> Two sets of colors have _different hue_ if the hue of any color in one set is different from the hue of any color in the other set.

Author: What I'm trying to convey is two sets have different hue if I can find one pair composed by a color from set 1 and set 2 where the two colors have a different hue. 
Using "each" instead of "any" my interpretation is that would have to be true for every pair of colors instead of one pair of colors.
@EmmaJP do you want to chime in?

> Two sets of colors have _different hue_ if the hue of any color in one set is different from the hue of any color in the other set.

Reviewer: I do understand the current text the same way as you 😀 (one pair is different means the set are different).

We can probably make that interpretation more obvious:
```suggestion
Two sets of colors have _different hue_ if the corresponding sets of hues are different.
```
(using set equality in the way)

or
```suggestion
Two sets of colors have _different hue_ if there is at least one color from the first set having a hue different from at least one color in the second set.
```

> Two sets of colors have _different hue_ if the hue of any color in one set is different from the hue of any color in the other set.

Reviewer 2: We use the word `hue` extensively. Can we link to a basic definition from color theory.

Author: I would like that also. Hue is defined here: http://cie.co.at/publications/colour-appearance-model-colour-management-systems-ciecam02
However, from what I can understand that document is only accessible to members of the ICI or something. What I learned is that it's not public. So, I'm not really sure what we can do. Any suggestion?

Reviewer 2: The above link is not accessible to me. (non ICI member). I went through other docs like MDN and color theory etc., no one is cross linking to what hue means, so we should be good.

> Two sets of colors have _different hue_ if the hue of the colors in one set is different from the hue of the colors in the other set.

Reviewer: When comparing sets, do you mean "pairwise different" (no element in common) or "globally different" (at least one element different)?
